osutil/sync_dir.go

```diff
@@ -43,6 +43,10 @@ var errSameState = fmt.Errorf("file state has not changed")
 // removed. Missing files are created and differing files are corrected.  Files
 // not matching the pattern are ignored.
 //
+// Note, that EnsureDirState() only checks for permissions and content. Other
```

Comment:
`Note that EnsureDirState only ...` (no parenthesis on method or function names, and no comma there)